### DISCUSSION TITLE
OK so hear me out...I think clown isn't as funny anymore...Lets fix it, and I have some ideas!

### DISCUSSION THREAD

**Discussion by ExLegendary** [FakeFishGames/Barotrauma#10816] [Ideas and suggestions]
So first you need to get rid of the clown crate....Shit is useless.

Here is how I envision the clown perks....

Perk 1

Choice between 30% more rewards and payouts for Honkmother quests....But have a 1% chance for it to be filled with explosives that detonate when crate is opened and double the available Honkmother quests in stations as more clowns approach you.

OR...

Bozo'ed which allows clown to craft whipped cream pies from INCREDIBLY easy grab materials that he can throw that have a 20% chance to stun for 4 seconds (this is due to the entity being so upset they are litterally shaking fr fr!). Throwing speed and arc is the same as grenades.....Does not work in water.

Perk 2

(Clownin Around) replaces Water Prankster....This perk allows the clown to swim outside with 100% movespeed!!! BUT!!! With NO suit and yes even crush depth you can swim...BUT...They cannot wear ANY suit but the clown mask nor armor! Your funny has it's price after all and your squishy flesh changed by the grace of the Honkmother forbids hiding your natural gellatin like body from the world!
(You also get -25% all physical damage)

Perk 3

Honkolocation.

If you honk your horn you can see in the dark for your entire screen's width outside in the water. "You do not need light sources for your highly trained ears can find the enemy before they see you!...If they also didn't hear you of course!" This also extends your field of view and allows you to see through walls for a VERY short time to identify hostiles behind it. Talking half-second.

Other option?

Honkmother protects!

This perk allows the clown to after being attacked and heavilly damaged by monsters to survive the killing blow and are left with 1hp and are cleared of all statuses and begin to ragdoll for 5 second window during which they are completely immune to damage but ALL hostiles in the area are IMMEDIATELY attracted to you and begin attacking as you look JUST SO DELICIOUS....Hope your friends want to save you!

FINAL PERK

Last laugh.

The big one...Honestly I want the clown to be funny but scary as a clown should be! SO! How best to do that? Oh your clown just ate a nuke....Yep. Thats it. Your clown is now a Ha-Ha-Hadrogen bomb! Though this only goes off when they die so it's used in conjunction with the other perk or not is up to you....Nukes are expensive after all and having that extra crewmate might actually be more of a loss than you might think! Reapers tax doesn't apply to this death too! BUT! On the chance you forget you are armed you can obliterate your own submarine so take care! You can't just vomit it up once you swallow those things!

**Comment by Arctodites** [comment]
I don't know about all of these perk changes, but whipped cream pies should definitely be a clown faction item, and honkolocation is a brilliant idea!

What else are clowns known for?  Maybe confetti cannons or glitter bombs?  Trampolines?  Lion tamer's hoop and whip?  I would love for there to be a Europan clown circus with side missions about capturing rare creatures for the show.  I feel like Barotrauma is definitely missing a 'catch-em-all' gameplay loop -- whether that be display aquariums or a Europan zoo.
